src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -23,34 +20,35 @@ public class Line extends BaseEntity {
 
     private Long downStationId;
 
-    private Long distance;
+    @Embedded
+    Sections sections = new Sections();
 
-    public Line() {
+    protected Line() {
 
     }
 
     public Line(String name) {
         this.name = name;
     }
 
-    public Line(String name, String color, Long upStationId, Long downStationId, Long distance) {
+    public Line(String name, String color, Long upStationId, Long downStationId) {
```

Comment:
```suggestion
    public Line(String name, String color, Station upStation, Station downStation, int distance) {
```
이렇게 파라미터를 받은 후
생성자 내부에서 `addSection(new Section(this, upStation, downStation, distance));`를 호출하는 방식으로 해보는건 어떤가요? 😀

`Line`의 `upStationId`, `downStationId`는 용도가 없으니 제거해도 될것같아요.

Reply:
위에 알려주신 방식 기준으로 수정 완료 했습니다~